gem_metrics/texts.py

```diff
@@ -13,72 +15,119 @@ class Texts:
 
     PUNCTUATION = set(string.punctuation)
 
-    def __init__(self, key, data):
-        self.key = key
+    def __init__(self, data_key, data, language="en"):
+        self.data_key = data_key
+        # TODO allow other data formats.
         if not isinstance(data, dict):
             self.filename = data
-            # TODO allow other data formats?
-            with open(data, 'r', encoding='UTF-8') as fh:
+            with open(data, "r", encoding="UTF-8") as fh:
                 data = json.load(fh)
+                self.all_data = data
         else:
-            self.filename = data.get('filename')
-        logger.info(f'Loading {self.__class__.__name__.lower()} for {str(self.filename)}')
-        self.all_data = data['values']
-
-        if len(data['language']) > 3 or data['language'][0].isupper():
-            self.language = languages.get(name=data['language'])
-        elif len(data['language']) == 3:
-            self.language = languages.get(alpha_3=data['language'])
-        else:
-            self.language = languages.get(alpha_2=data['language'])
-
-        # allow bare lists of strings as well as lists of dicts
-        if self.all_data and isinstance(self.all_data[0], str):
+            self.filename = data.get("filename")
+            self.all_data = data["values"]
+        logger.info(
+            f"Loading {self.__class__.__name__.lower()} for {str(self.filename)}"
+        )
+
+        self.language = languages.get(alpha_2=language)
```

Comment:
Any specific reason to remove the flexibility here (having language="English" worked the same as "eng" and "en", now it doesn't)?

Reply:
Now it is set globally in the `config.py` and is no longer required in the individual submission files. Hence the simplification.

Reply:
OK, makes sense :-)